make-options.json

```diff
@@ -16,6 +16,7 @@
         "AzureCloudPowerShellDeploymentV2",
         "AzureContainerAppsV0",
         "AzureContainerAppsV1",
+		"AzureDevOpsTestPlanV0",
```

Comment:
Could you try add the task name to a Node20 section and rebuild it locally?

Reply:
Added and built code locally.